mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -736,7 +736,7 @@ class spawn_point_types {
 	class arvn_grnd_firesupport_light_custom {
```

Comment:
no extra arvn statics spawner? John --> 😢

Reply:
uhhh.... next update

Reply:
this PR, or a separate PR?

Reply:
separate that I will do another time.

Reply:
forgot your answers from a few days ago.

Reply:
ok.

(btw i'm loving the git log giving me notifications about PR things without having to look at my email 🎉 )